lib/Backends/OpenCL/OpenCLDeviceManager.h

```diff
@@ -164,6 +164,10 @@ class OpenCLDeviceManager : public QueueBackedDeviceManager {
   /// The TID of the device (for TraceEvents);
   int deviceTid_{-1};
 
+  /// Map from PH to functionName for static placeholders.
```

Comment:
Most of the OpenCL DeviceManager keys it's structures by functionName so it's more convenient. Since the OpenCLCompiledFunction keeps a reference to Function* we could use that instead but this seems cleaner.